libraries/AP_ADSB/AP_ADSB_uAvionix_MAVLink.cpp

```diff
@@ -38,13 +38,18 @@ bool AP_ADSB_uAvionix_MAVLink::detect()
 
 void AP_ADSB_uAvionix_MAVLink::update()
 {
+    // if we are an in only instance then we are done here, the work below is all related to sending outputs
+    if (_frontend.get_type(_instance) == AP_ADSB::Type::uAvionix_MAVLink_In) {
+        return;
+    }
+
     const uint32_t now = AP_HAL::millis();
 
     // send static configuration data to transceiver, every 5s
     if (_frontend.out_state.chan_last_ms > 0 && now - _frontend.out_state.chan_last_ms > ADSB_CHAN_TIMEOUT_MS) {
         // haven't gotten a heartbeat health status packet in a while, assume hardware failure
-        // TODO: reset out_state.chan
         _frontend.out_state.chan = -1;
+        _frontend.out_state.chan_last_ms = 0; // if the time isn't reset we spam the message
```

Comment:
separate PR so we can backport to 4.4/4.3

Reply:
Separate PR here #24150